SMPP/SmppTestClient/SMSControl.cs

```diff
@@ -53,7 +53,7 @@ static void Main(string[] args)
                 // Hit Enter in the terminal once the binds are up to see this prompt
 
                 Console.WriteLine("Commands");
-                Console.WriteLine("send 12223334444 hello jack");
+                Console.WriteLine("send 5551991096510 hello");
```

Comment:
Not sure what this number is but it is now published. If it is something you don't want others to know about, please change. it.